test/fixtures/mod-deps/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/gorilla/mux"
```

Comment:
Hmm...interesting. How did this test pass? Without specifying GO111MODULE=on, Go1.11rc2 doesn't fetch deps for me.

Reply:
Are you using modules inside of a GOPATH? If so that's why. AFAICT, outside of a GOPATH, modules are turned on.

Reply:
Ah, that's true. I'm not sure I follow the rationale behind this (I would expect it's either on or off). So I assume GOPATH hasn't been set in our case that's why the test passes?

Reply:
Yes, when a go.mod file is detected, we don't construct a $GOPATH, we just figure some stuff out and build the code. If this hold true I can't wait for the day when I can remove all the other tools. :-)

Reply:
❤️